example_helper_test.go

```diff
@@ -17,7 +17,7 @@ func httpLogHelper(ctx context.Context, status int) {
 	)
 }
 
-var l = sloghuman.Make(os.Stdout)
+var l = slog.Make(sloghuman.Make(os.Stdout))
```

Comment:
Having the same function name for creating a sink in a logger now that they're separate seems weird to me. Maybe renaming the sink constructors to `Sink(...)` would be a bit clearer.

Reply:
Oh I like that. Changing now.